### PR TITLE
Add receiving state to shipment table

### PR DESCRIPTION
**Pull request by HaGuesto**

### CODE REVIEW

**Comment by HaGuesto** [comment]
[Add receiving_started_on and receiving_started_by fields to shipments db-table](https://trello.com/c/omJaOa2A/1131-add-receivingstartedon-and-receivingstartedby-fields-to-shipments-db-table)

**Review by pylipp — APPROVED** [review]

File: react/src/views/Transfers/ShipmentsOverview/ShipmentsOverviewView.tsx [hunk 1]
```diff
@@ -79,6 +79,7 @@ function ShipmentsOverviewView() {
         const shipmentUpdateDateTimes = [
           element.startedOn,
           element.sentOn,
+          element.receivingStartedOn,
```
Comment (HaGuesto) [inline react/src/views/Transfers/ShipmentsOverview/ShipmentsOverviewView.tsx:82]: good. was afraid u miss that @pylipp

**Comment by HaGuesto** [comment]
From my side we can merge this @pylipp

File: react/src/queries/fragments.ts [hunk 2]
```diff
@@ -275,6 +275,10 @@ export const SHIPMENT_FIELDS_FRAGMENT = gql`
     sentBy {
       ...UserBasicFields
     }
+    receivingStartedOn
```
Comment (HaGuesto) [inline react/src/queries/fragments.ts:278]: fyi @pylipp we did not update the mock for test 4.4.1.4. Therefore, the test fails now.

I pushed a quick fix on master

Comment (pylipp) [reply react/src/queries/fragments.ts:278]: Ooh thanks :+1:

**Pull request merged**